app/controllers/DraftMetadataUploadController.scala

```diff
@@ -35,4 +42,30 @@ class DraftMetadataUploadController @Inject() (
       }
     }
   }
+
+  def saveDraftMetadata(consignmentId: java.util.UUID): Action[MultipartFormData[TemporaryFile]] = secureAction.async(parse.multipartFormData) {
+
+    implicit request: Request[MultipartFormData[TemporaryFile]] =>
+      val successPage = routes.DraftMetadataChecksController.draftMetadataChecksPage(consignmentId)
+      val token = request.asInstanceOf[Request[AnyContent]].token
+
+      val uploadBucket = s"${applicationConfig.draft_metadata_s3_bucket_name}-${applicationConfig.frontEndInfo.stage}"
```

Comment:
The full bucket name is held in the draft_metadata_s3_bucket_name environment variable, so this need to be: `val uploadBucket = s"applicationConfig.draft_metadata_s3_bucket_name"`

See the Frontend ECS task definition here: https://eu-west-2.console.aws.amazon.com/ecs/v2/task-definitions/frontend-intg/63/json?region=eu-west-2